tests/osaka/eip7692_eof_v1/eip5450_stack/test_code_validation.py

```diff
@@ -433,3 +433,56 @@ def test_rjumps_jumpf_nonreturning(
         possible_exceptions.append(EOFException.INVALID_NON_RETURNING_FLAG)
 
     eof_test(container=Container(sections=sections), expect_exception=possible_exceptions or None)
+
+
+@pytest.mark.parametrize(
+    "container",
+    [
+        Container(
+            name="underflow_0",
+            sections=[
+                Section.Code(
+                    code=Op.ADD + Op.STOP,
```

Comment:
```suggestion
                    code=Op.POP + Op.STOP,
```
We could also add another test that uses an opcode that barely produces the underflow vs one that underflows by two items (and pushes one).

`max_stack_height` could also use more variants here, e.g. 1 and 2 in this case, and 0 and 1 in the case of `Op.POP` for example.

Reply:
> The exhaustive test should take all opcodes with static non-zero stack height required?

ah yes, zero-stack height required cannot overflow. I've meant to say you can either test underflow with 0 items or with `< inputs` items on stack

Reply:
These has been added in https://github.com/ethereum/execution-spec-tests/pull/1378.